src/cloud/architecture/scaled-architecture.md

```diff
@@ -46,7 +46,7 @@ Minimally, Pro projects with the Scaled architecture have 6 nodes available.
 -  3 web nodes c5.2xlarge (8 CPU, 16 Gb RAM)
 -  3 core nodes m5.2xlarge (8 CPU, 32 Gb RAM)
 
-Each project is unique, however, and requires performance monitoring to properly analyze resource management. Each account includes the [New Relic Infrastructure service][nri], which automatically connects with the application data and performance analytics to provide dynamic server monitoring. As a resource reaches capacity or a degradation in performance, you can create a request to scale your infrastructure to meet the demand.
+Each project is unique, however, and requires performance monitoring to properly analyze resource management. Each account includes the [New Relic Infrastructure service][nri], which automatically connects with the application data and performance analytics to provide dynamic server monitoring. Specifically, you can use the New Relic service to monitor CPU and RAM utilization to determine which nodes require additional resources. As a resource reaches capacity or a degradation in performance, you can create a request to scale your infrastructure to meet the demand.
```

Comment:
```suggestion
Each project is unique, however, and requires performance monitoring to properly analyze resource management. Each account includes the [New Relic Infrastructure service][nri], which automatically connects with the application data and performance analytics to provide dynamic server monitoring. Specifically, you can use the New Relic service to monitor CPU and RAM utilization to determine which nodes require additional resources. As a resource reaches capacity or experiences a degradation in performance, you can create a request to scale your infrastructure to meet the demand.
```

Reply:
Resources don't experience things. Users do. But I will fix this because it isn't quite right. Thanks!